lib/Notes/NoteCreatePage/NoteCreatePage.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router';
+
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes-core';
+import { Icon } from '@folio/stripes-components';
+
+import NoteForm from '../NoteForm';
+import noteTypesCollectionShape from '../response-shapes';
+
+@stripesConnect
+class NoteCreatePage extends Component {
+  static propTypes = {
+    domain: PropTypes.string.isRequired,
+    entityTypeTranslationKeyMap: PropTypes.objectOf(PropTypes.string),
+    mutator: PropTypes.object,
+    paneHeaderAppIcon: PropTypes.string.isRequired,
+    previousLocation: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    referredEntityData: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      type: PropTypes.string.isRequired,
+    }).isRequired,
+    resources: PropTypes.shape({
+      noteTypesData: noteTypesCollectionShape
+    }),
+  }
+
+  static manifest = {
+    noteTypesData: {
+      type: 'okapi',
+      path: 'note-types',
+    },
+    notes: {
+      type: 'okapi',
+      path: 'notes',
+      fetch: false,
+    },
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  state = {
+    submitIsPending: false,
+    submitSucceeded: false,
+  }
+
+  async onSubmit(noteData) {
```

Comment:
```suggestion
 onSubmit = async noteData => {
```
so you would get rid of constructor.